style.css

```diff
@@ -928,8 +2035,158 @@ body {
     top: 614px;
   }
 
+ .second__wrap {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)